### DISCUSSION TITLE
Version of serin used by 5.2.3?

### DISCUSSION THREAD

**Discussion by alexatcano** [serenity-is/Serenity#7138] [Help]
Hello,

I inherited a 5.2.3 version project and I am wondering what the corresponding version of the serin npm package was used.

`$ npm view sergen versions
`

yields

`
[ '2.8.601', '2.8.602', '2.9.300', '2.9.2100', '2.9.2101' ]`

also
`npm view serin versions`

yields
`[ '1.0.0',
  '1.0.1',
  '1.0.2',
  '1.0.3',
  '2.8.701',
  '2.8.702',
  '2.8.800',
  '2.8.900',
  '2.8.1000',
  '2.8.1001',
  '2.8.1002',
  '2.8.1100',
  '2.9.2100',
  '3.3.1400',
  '3.3.1500',
  '3.3.1501',
  '3.4.0',
  '3.5.0',
  '6.3.6',
  '6.4.3' ]`

I checked the corresponding documentation https://github.dev/serenity-is/Serenity/tree/8abaeca869624acc75907ecb4b5eee7bee38355c, but I can only guess 3.5.0 should be ok?

any help would be great.

**Comment by VictorTomaili** [comment]
Why do you need to use serin?

As of Serene 6.4.4, the old serin tool is obsolete and may not work properly to create new Serene projects as we are now using dotnet new templating.

https://serenity.is/Docs/getting_started/installing_serene_with_dotnet_new#:~:text=Note%3A%20As%20of%20Serene%206.4.4%2C%20the%20old%20serin%20tool%20is%20obsolete%20and%20may%20not%20work%20properly%20to%20create%20new%20Serene%20projects%20as%20we%20are%20now%20using%20dotnet%20new%20templating.

**Reply by alexatcano** [reply]
No particular reason other than following the 5.2.3 steps. If can use the templating with the old 5.2.3, that's great.  Why not update to the latest? It's not up to me.

**Reply by VictorTomaili** [reply]
the serin is a npm version of the template to use in linux. With the latest updates, linux have dotnet new command and serin not required anymore. You can use the dotnet new instead to use serene latest version

![image](https://github.com/serenity-is/Serenity/assets/2023440/36335af8-bbd6-4bda-aea6-72c40b6cc8cb)

**Reply by alexatcano** [reply]
Thanks Victor, definitely updating to the latest sounds reasonable. But, since this is an old working production app using the pro version, we are not allowed to update at the moment.   Stuck with .NET 6.0 and serene 5.2.3 pro.

**Reply by VictorTomaili** [reply]
Serin is utilized for the free Serene template, not for StartSharp. It fetches the Serene template from the Visual Studio Marketplace to create the project. To create a StartSharp project, download the 5.2.3 VSIX file from the members' area and use Visual Studio. Afterward, you can continue with Linux or Mac.
